filebeat/module/postgresql/log/test/postgresql-13-error-code.csv.log-expected.json

```diff
@@ -0,0 +1,145 @@
+        "@timestamp": "2021-02-14T10:45:48.113Z",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "postgresql.log",
+        "event.kind": "event",
+        "event.module": "postgresql",
+        "event.timezone": "UTC",
+        "event.type": [
+            "info"
+        ],
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.level": "LOG",
+        "log.offset": 224,
+        "message": "BEGIN;",
+        "postgresql.log.application_name": "psql",
+        "postgresql.log.backend_type": "client backend",
+        "postgresql.log.client_addr": "172.24.0.1",
+        "postgresql.log.client_port": 48978,
+        "postgresql.log.command_tag": "idle",
+        "postgresql.log.database": "postgres",
+        "postgresql.log.session_id": "6028ff3a.56",
+        "postgresql.log.session_start_time": "2021-02-14T10:45:14.000Z",
+        "postgresql.log.sql_state_code": "00000",
+        "postgresql.log.timestamp": "2021-02-14 10:45:48.113 UTC",
+        "postgresql.log.transaction_id": 0,
+        "postgresql.log.virtual_transaction_id": "3/4",
+        "process.pid": "86",
+        "related.user": [
+            "postgres"
+        ],
+        "service.type": "postgresql",
+        "user.name": "postgres"
+    },
+    {
+        "@timestamp": "2021-02-14T10:45:48.164Z",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "postgresql.log",
+        "event.kind": "event",
+        "event.module": "postgresql",
+        "event.timezone": "UTC",
+        "event.type": [
+            "error"
+        ],
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.level": "FATAL",
+        "log.offset": 409,
+        "message": "terminating connection due to idle-in-transaction timeout",
+        "postgresql.log.application_name": "psql",
+        "postgresql.log.backend_type": "client backend",
+        "postgresql.log.client_addr": "172.24.0.1",
+        "postgresql.log.client_port": 48978,
+        "postgresql.log.command_tag": "idle in transaction",
+        "postgresql.log.database": "postgres",
+        "postgresql.log.session_id": "6028ff3a.56",
+        "postgresql.log.session_start_time": "2021-02-14T10:45:14.000Z",
+        "postgresql.log.sql_state_code": "25P03",
+        "postgresql.log.timestamp": "2021-02-14 10:45:48.164 UTC",
+        "postgresql.log.transaction_id": 0,
+        "postgresql.log.virtual_transaction_id": "3/4",
+        "process.pid": "86",
+        "related.user": [
+            "postgres"
+        ],
+        "service.type": "postgresql",
+        "user.name": "postgres"
+    },
+    {
+        "@timestamp": "2021-02-14T10:45:48.164Z",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "postgresql.log",
+        "event.kind": "event",
+        "event.module": "postgresql",
+        "event.timezone": "UTC",
+        "event.type": [
+            "info"
+        ],
```

Comment:
We may need to regenerate these files after the change from `append` to `set`.

Reply:
Yeah, thats odd, I noted that too, for some reason they didn't failed

Reply:
```
 make update && GENERATE=1 INTEGRATION_TESTS=1 BEAT_STRICT_PERMS=false TESTING_FILEBEAT_MODULES=postgresql pytest tests/system/test_modules.py
```
This command didn't do the trick. :confused:

Reply:
Running tests locally I see this error that seems related:
```
Invalid [path] value [postresql.log.sql_state_code] for field alias [postgresql.log.error.code]: an alias must refer to an existing field in the mappings.
```

Also, a unexpected collateral effect, tests in `filebeat/scripts/tester/main_test.go` have started to fail because they use the postgresql pipelines as data files :grimacing: Could you update this test?

Reply:
Oh, just saw here. The `filebeat/scripts/tester/main_test.go` I think that I've fixed, Now I'm looking at the alias.

Reply:
All green now :) thanks!